packages/fints/src/client.ts

```diff
@@ -35,6 +37,18 @@ export abstract class Client {
         ]));
         await dialog.end();
         const hispa = response.findSegment(HISPA);
+
+        hispa.accounts.map((account) => {
+            const hiupdAccount = dialog.hiupd.filter((element) => {
+                return (element.account.iban === account.iban);
+            });
+            if (hiupdAccount.length > 0) {
+                account.accountOwnerName = hiupdAccount[0].account.accountOwnerName1;
```

Comment:
Instead of assigning to `account` here, mutating the input data, return a new object:

```
{
    ...acccount,
    accountOwnerName: hiupdAccount[0].account.accountOwnerName,
    accountName: hiupdAccount[0].account.accountName,
    limitValue: Parse.num(hiupdAccount[0].account.limitValue),
}
```

and the return the result of the `.map`.